### PR TITLE
[zh-cn]: update translation of `console.time()` static method

### PR DESCRIPTION
**Pull request by T34-active**

### CODE REVIEW

File: files/zh-cn/web/api/console/time_static/index.md [hunk 1]
```diff
@@ -31,8 +35,10 @@ time(label)
...
-## 参阅
+## 参见
 
-- {{domxref("Console.timeEnd()")}}
-- {{domxref("Console.timeLog()")}}
-- [Opera Dragonfly documentation: Console](http://www.opera.com/dragonfly/documentation/console/)
+- {{domxref("console/timeEnd_static", "console.timeEnd()")}}
+- {{domxref("console/timeLog_static", "console.timeLog()")}}
+- [微软 Edge 浏览器关于 `console.time()` 的文档](https://learn.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/console/api#time)
+- [Node.JS 浏览器关于 `console.time()` 的文档](https://nodejs.org/docs/latest/api/console.html#consoletimelabel)
```
Comment (T34-active) [inline files/zh-cn/web/api/console/time_static/index.md:43]:
```suggestion
- [Node.JS 关于 `console.time()` 的文档](https://nodejs.org/docs/latest/api/console.html#consoletimelabel)
```

File: files/zh-cn/web/api/console/time_static/index.md [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
 
-{{APIRef("Console API")}}
+{{APIRef("Console API")}} {{AvailableInWorkers}}
 
 你可以启动一个计时器来跟踪某一个操作的占用时长。每一个计时器必须拥有唯一的名字，页面中最多能同时运行 10,000 个计时器。当以此计时器名字为参数调用 {{ domxref("console.timeEnd()") }} 时，浏览器将以毫秒为单位，输出对应计时器所经过的时间。
 
-关于 [Timers](/zh-CN/DOM/console#Timers) 的细节和例子请参考文档 {{ domxref("console") }} 。
-
-{{AvailableInWorkers}}
+关于 [Timers](/zh-CN/DOM/console#Timers) 的细节和例子请参考文档 {{ domxref("console") }}。
```
Comment (T34-active) [inline files/zh-cn/web/api/console/time_static/index.md:12]:
```suggestion
关于 [Timers](/zh-CN/DOM/console#Timers) 的细节和示例请参考文档 {{ domxref("console") }}。
```

**Review by T34-active — COMMENTED** [review]
fix content

File: files/zh-cn/web/api/console/time_static/index.md [hunk 3]
```diff
@@ -1,15 +1,15 @@
...
 
-{{APIRef("Console API")}}
+{{APIRef("Console API")}} {{AvailableInWorkers}}
 
 你可以启动一个计时器来跟踪某一个操作的占用时长。每一个计时器必须拥有唯一的名字，页面中最多能同时运行 10,000 个计时器。当以此计时器名字为参数调用 {{ domxref("console.timeEnd()") }} 时，浏览器将以毫秒为单位，输出对应计时器所经过的时间。
 
-关于 [Timers](/zh-CN/DOM/console#Timers) 的细节和例子请参考文档 {{ domxref("console") }} 。
-
-{{AvailableInWorkers}}
+关于 [Timers](/zh-CN/DOM/console#Timers) 的细节和示例请参考文档 {{ domxref("console") }}。
```
Comment (yin1999) [inline files/zh-cn/web/api/console/time_static/index.md:12]:
```suggestion
关于[计时器](/zh-CN/docs/Web/API/console#计时器)的细节和示例请参考文档 {{ domxref("console") }}。
```

**Review by yin1999 — APPROVED** [review]
感谢 @T34-active

**Pull request merged**